src/systems/a320_systems_wasm/src/lib.rs

```diff
@@ -272,12 +272,16 @@ impl MsfsAspectCtor for Flaps {
                 right_slat: 0.,
             },
 
-            flaps_handle_position: 0,
-            left_flaps_position: 0.,
-            right_flaps_position: 0.,
-
-            left_slats_position: 0.,
-            right_slats_position: 0.,
+            flaps_handle_position: NamedVariable::from("A32NX_FLAPS_HANDLE_INDEX").get_value(),
+            left_flaps_position: NamedVariable::from("A32NX_LEFT_FLAPS_POSITION_PERCENT")
+                .get_value(),
+            right_flaps_position: NamedVariable::from("A32NX_RIGHT_FLAPS_POSITION_PERCENT")
+                .get_value(),
+
+            left_slats_position: NamedVariable::from("A32NX_LEFT_SLATS_POSITION_PERCENT")
+                .get_value(),
+            right_slats_position: NamedVariable::from("A32NX_RIGHT_SLATS_POSITION_PERCENT")
+                .get_value(),
```

Comment:
This will collide with #6427. Please wait for that one to be merged, as I have somewhat less time in the coming period to resolve things.

Reply:
Actually you'll be happy to know that these fields will no longer exist. Reads and writes will happen directly on the named variable itself.

Reply:
So I should just remove changes to this file in this PR, and wait for that PR to be merged?

Reply:
Yep. That's right. I think that should work!

Reply:
Done!